lib/pharos/phases/configure_calico.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Phases
+    class ConfigureCalico < Pharos::Phase
+      title "Configure Calico network"
+
+      CALICO_VERSION = '3.1.0'
+
+      register_component(
+        Pharos::Phases::Component.new(
+          name: 'calico-node', version: CALICO_VERSION, license: 'Apache License 2.0'
+        )
+      )
+
+      register_component(
+        Pharos::Phases::Component.new(
+          name: 'calico-cni', version: CALICO_VERSION, license: 'Apache License 2.0'
+        )
+      )
+
+      def validate
+        case @host.cpu_arch.name
+        when 'amd64'
+
+        else
+          fail "Unsupported CPU architecture: #{@host.cpu_arch.name}"
```

Comment:
Calico for ARM is still WiP: https://github.com/projectcalico/calicoctl/issues/1475

> So far we have 4 components on multiple archs, 1 with an open PR, and more to come

Although it sounds like they're aiming for multi-arch images, so we shouldn't need to template it for the manifests: 

> Single image tag with multi-arch manifests is following and in process.